code/modules/surgery/implant.dm

```diff
@@ -199,7 +199,6 @@
 				var/obj/item/implant/imp = I
 				user.visible_message("<span class='warning'>Something beeps inside [target]'s [affected.display_name]!</span>")
 				playsound(imp.loc, 'sound/items/countdown.ogg', 25, 1)
-				spawn(25)
-					imp.activate()
+				addtimer(CALLBACK(imp, /obj/item/implant.proc/activate), 25)
```

Comment:
```suggestion
				addtimer(CALLBACK(imp, /obj/item/implant.proc/activate), 2.5 SECONDS)
```

Reply:
pretty sure this is the length of the sound in the line above.

Reply:
Make it a define and undef it right after the proc?

Reply:
spawn(x) is x/10 seconds but addtimer is in ticks so they arent really equiv.